src/jet/main.clj

```diff
@@ -94,7 +94,9 @@
                 :interactive :collect
                 :edn-reader-opts
                 :help]} (parse-opts args)]
-    (cond version
+    (if (nil? args)
+      (print-help)
+      ((cond version
```

Comment:
This is a typo probably. ((cond ...) should be (cond ...)

Reply:
Also, I noticed that one of the tests, the `interactive-test` had the no-args as a test case. Would we have to make a change to that?